types/service/database_service_configuration.go

```diff
@@ -216,8 +216,8 @@ type StandardDatabaseServiceConfiguration struct {
 
 	UsernameAndPasswordAuth *DatabaseUsernameAndPasswordAuthConfiguration `json:"username_and_password_auth_configuration,omitempty"`
 	TlsAuth                 *DatabaseTlsAuthConfiguration                 `json:"tls_auth_configuration,omitempty"`
-	Kerberos                *UsernameAndPassword                          `json:"kerberos_configuration,omitempty"`
-	SqlAuthentication       *UsernameAndPassword                          `json:"sql_authentication_configuration,omitempty"`
+	Kerberos                *DatabaseUsernameAndPasswordAuthConfiguration `json:"kerberos_configuration,omitempty"`
```

Comment:
These are all the same type?

Reply:
If its the exact same data, i'd still wanna see it duplicated in a new struct (mostly for users of the SDK it will be more clear to use 

```
dbServiceConfig := &service.StandardDatabaseServiceConfiguration{
...
    Kerberos: &service.KerberosConfiguration{...}
...
}
```

than


```
dbServiceConfig := &service.StandardDatabaseServiceConfiguration{
...
    Kerberos: &service. DatabaseUsernameAndPasswordAuthConfiguration{...}
...
}
```

Reply:
done